test-app-async/src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagenameasync/integration/HmppsQueueResourceTest_asyncClient.kt

```diff
@@ -0,0 +1,120 @@
+package uk.gov.justice.digital.hmpps.hmppstemplatepackagenameasync.integration
+
+import org.awaitility.kotlin.await
+import org.awaitility.kotlin.matches
+import org.awaitility.kotlin.untilCallTo
+import org.hamcrest.Matchers
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.verify
+import org.springframework.http.MediaType
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest
+import uk.gov.justice.digital.hmpps.hmppstemplatepackagenameasync.service.EventType
+import uk.gov.justice.digital.hmpps.hmppstemplatepackagenameasync.service.HmppsEvent
+import uk.gov.justice.digital.hmpps.hmppstemplatepackagenameasync.service.Message
+import uk.gov.justice.digital.hmpps.hmppstemplatepackagenameasync.service.MessageAttributes
+
+class HmppsQueueResourceTest_asyncClient : IntegrationTestBase() {
```

Comment:
Added some integration tests for async SQS clients to both the non-reactive and reactive test apps.